features/account/settings.feature

```diff
@@ -0,0 +1,18 @@
+Feature: User settings page
```

Comment:
@ORIGIN-Ticket-007
Feature: User Settings Page
As a logged user, on the settings page, I want to be able to see the user profile details. The user should be able to update the information.